index.html

```diff
@@ -54,5 +56,250 @@ <h1 class="name">
+              sign-ups required.
+            </p>
+          </div>
+          <!--  -->
+    
+          <div>
+            <ul class="tech-Stack">
+              <li id="html">HTML</li>
+              <li id="css">CSS</li>
+              <li id="js">JAVASCRIPT</li>
+            </ul>
+          </div>
+    
+          <!--  -->
+          <div><button type="button" class="seeProject">see project</button></div>
+    
+        </div>
+    </div>
+</section>
+
+<!-- about me section -->
+<div class="cover"> 
+<section class="about-me">
+
+    <div class="myself">
+        <div class="">
+          <h2 class="headSelf">
+            About Myself
+          </h2>
+
+          <p class="text">
+            Hello I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+
+          <p class="connect">
+            LET'S CONNECT
+           </p>
+           
+           <div class="social-media">
+               <ul> 
+                   <li><a href="" id="twitter"><img src="images/twitter.png" alt=""></a> </li>
+                   <li><a href="" id="linkin" ><img src="images/Linkedin icon.png" alt=""></a> </li> 
+                   <li><a href="" id="medium"><img src="images/medium.png" alt=""></a> </li>
+                   <li><a href="" id="github"><img src="images/github.png" alt=""></a></li>
+                   <li><a href="" id="angelist"><img src="images/angellist.png" alt=""></a></li>
+               </ul>
+           </div>
+          <p><button type="button" class="about2">Get my resume</button></p>
+        </div>
+<!--  -->
+        <ul class="languages">
+          <li class="">
+            <div class="arrow">
+              <h3 class="lan">Languages</h3>
+              <img class="arrowsign" src="images/Union.png" alt="" />
+            </div>
+
+            <div class="">
+              <img src="/images/Frame 9.png" alt="image" />
+              <img src="/images/Frame 10.png" alt="image" />
+              <img src="/images/Frame 11.png" alt="image" />
+            </div>
+          </li>
+          <li class="arrow">
+            <h3 class="lan">FRAMEWORKS</h3>
+            <p><img class="election2" src="./images/./left.png" alt="" /></p>
+          </li>
+          <li>
+            <img
+              class="separator"
+              src="images/Separator Bottom.png"
+              alt=""
+            />
+          </li>
+
+          <li class="arrow">
+            <h3 class="lan">Skills</h3>
+            <div>
+              <img class="election2" src="./images/./left.png" alt="" />
+            </div>
+          </li>
+
+          <li>
+            <img
+              class="separator"
+              src="/images/Separator Bottom.png"
+              alt=""
+            />
+          </li>
+        </ul>

```

Comment:
[Optional] The `ul` you used here, is not being properly used as the `li` elements are not related, like the sections and the divider lines, because they are separate groups of content and elements that are unrelated. Consider using instead a `div` to replace the `ul` element wrapping the rest of the content.

As for the `li` elements, replace them with an `ul` element as they are wrapping lists of related elements, as shown on the `languages` list

![image](https://user-images.githubusercontent.com/50066876/235490667-b07148c1-9f8a-4e56-983e-8211db7575f1.png)